manim/mobject/mobject.py

```diff
@@ -169,16 +171,17 @@ def remove(self, *mobjects):
     def get_array_attrs(self):
         return ["points"]
 
-    def digest_mobject_attrs(self):
-        """
-        Ensures all attributes which are mobjects are included
-        in the submobjects list.
-        """
-        mobject_attrs = [
-            x for x in list(self.__dict__.values()) if isinstance(x, Mobject)
-        ]
-        self.submobjects = list_update(self.submobjects, mobject_attrs)
-        return self
+    # This does not seem to be in use any more:
+    # def digest_mobject_attrs(self):
+    #     """
+    #     Ensures all attributes which are mobjects are included
+    #     in the submobjects list.
+    #     """
+    #     mobject_attrs = [
+    #         x for x in list(self.__dict__.values()) if isinstance(x, Mobject)
+    #     ]
+    #     self.submobjects = list_update(self.submobjects, mobject_attrs)
+    #     return self
```

Comment:
Issue: please don't add commented out code to the codebase. If you mean to delete the code, just delete it :slightly_smiling_face: 
```suggestion
```